Slambook/index.js

```diff
@@ -67,9 +67,32 @@ form.addEventListener('submit',function(e){
     allSlamArray.push(formValue)
     console.log(formValue)
 console.log(allSlamArray)
+showSlamList();
+})
```

Comment:
You are referencing 'allSlamArray' in the forEach loop, but 'allSlamArray' is not defined. It seems like you might have intended to use a different array here. Make sure to define and pass the correct array to the forEach loop to avoid errors.